node/resolvers/paths.ts

```diff
@@ -46,6 +46,8 @@ const paths = {
   updateItems: (account, data) => `${paths.addItem(account, data)}/update`,
 
   shipping: account => `http://${account}.vtexcommercestable.com.br/api/checkout/pub/orderForms/simulation`,
+  
+  skuById: (account, { skuId }) => `http://${account}.vtexcommercestable.com.br/api/catalog_system/pvt/sku/stockkeepingunitbyid/${skuId}`,
```

Comment:
Strange naming, you are passing the SkuId to get the SkuId? A better naming would be getSKUItems, I believe.

Reply:
Actually, I'm passing the skuId to retrieve the sku item itself. In other words, I'm getting the SKU by id. I think that this name is clear enough.

Reply:
usem catalog_url base `${paths.catalog(account)}/pvt/sku/stockkeepingunitbyid/${skuId}`